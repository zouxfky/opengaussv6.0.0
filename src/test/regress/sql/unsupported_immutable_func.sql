drop procedure procedure_name_immutable;

create or replace procedure procedure_name_immutable
IMMUTABLE
NOT SHIPPABLE NOT LEAKPROOF
STRICT EXTERNAL
SECURITY INVOKER PACKAGE
COST 480 ROWS 528
IS
begin
NULL;
rollback;
end;
/

