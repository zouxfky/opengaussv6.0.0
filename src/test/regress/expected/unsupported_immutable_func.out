drop procedure procedure_name_immutable;
ERROR:  function procedure_name_immutable does not exist
create or replace procedure procedure_name_immutable
IMMUTABLE
NOT SHIPPABLE NOT LEAKPROOF
STRICT EXTERNAL
SECURITY INVOKER PACKAGE
COST 480 ROWS 528
IS
begin
NULL;
rollback;
end;
/
NOTICE:  Immutable function will be shippable anyway.
ERROR:  ROWS is not applicable when function does not return a set
